networkx/linalg/bethehessianmatrix.py

```diff
@@ -10,63 +10,70 @@
 def bethe_hessian_matrix(G, r=None, nodelist=None):
     r"""Returns the Bethe Hessian matrix of G.
 
-    The Bethe Hessian is a family of matrices parametrized by r, defined as
-    H(r) = (r^2 - 1) I - r A + D where A is the adjacency matrix, D is the
-    diagonal matrix of node degrees, and I is the identify matrix. It is equal
-    to the graph laplacian when the regularizer r = 1.
+     The Bethe Hessian is a family of matrices parametrized by r, defined as
+     H(r) = (r^2 - 1) I - r A + D where A is the adjacency matrix, D is the
+     diagonal matrix of node degrees, and I is the identify matrix. It is equal
+     to the graph laplacian when the regularizer r = 1.
```

Comment:
Looks like the indentation issue is still there, you can look at the current rendered docs at 

https://output.circle-artifacts.com/output/job/c728b112-dc65-4ae4-8eeb-5704158def20/artifacts/0/doc/build/html/reference/generated/networkx.linalg.bethehessianmatrix.bethe_hessian_matrix.html

There should be these additional spaces here. The documentation rendered created this as a quote instead of normal function documentation.

<img width="746" alt="Screenshot 2022-04-04 at 00 00 52" src="https://user-images.githubusercontent.com/5363860/161446037-f5da43b7-48a3-49b9-b14f-faaf0d3c2883.png">

The following change should fix it

```suggestion
    The Bethe Hessian is a family of matrices parametrized by r, defined as
    H(r) = (r^2 - 1) I - r A + D where A is the adjacency matrix, D is the
    diagonal matrix of node degrees, and I is the identify matrix. It is equal
    to the graph laplacian when the regularizer r = 1.
```